### DISCUSSION TITLE
Way to export results of a vote?

### DISCUSSION THREAD

**Discussion by Icedcool** [snapshot-labs/snapshot#2167] [Q&A]
We recently had a vote in our snapshot, and I'm interested in exporting the results for analysis.

Is there a way to do that?
Our snapshot:
https://snapshot.org/#/banklessvault.eth

**Comment by samwhosung** [comment]
There was an option to download a CSV, we disabled it because it was not working on all proposals, some proposal with large amount of votes wouldn't work. I'd recommend to use the GraphQL API to query and generate reports https://docs.snapshot.org/graphql-api

**Reply by bgoblirsch** [reply]
I am trying to use this to export the results of our vote, but I am getting 
`{
  "data": {
    "proposal": null
  }
}`

I am able to query older vote results just fine though. The vote just ended on Friday April 15, do I need to allow more time before I can access the data?

Here is one of the proposal IDs that is not accessible:
QmZikDmMvr4UzbJmvh7nCHmuGt4bxEAhnhZgonSsrpH4Ty

**Reply by metrox-eth** [reply]
Can you retry using the proposal hash that you can find at the end of the URL of the proposal ?

**Comment by Icedcool** [comment]
Thank you!

On Wed, Apr 13, 2022 at 9:13 AM Sam ***@***.***> wrote:

> There was an option to download a CSV, we disabled it because it was not
> working on all proposals, some proposal with large amount of votes wouldn't
> work. I'd recommend to use the GraphQL API to query and generate reports
> https://docs.snapshot.org/graphql-api
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/snapshot-labs/snapshot/discussions/2167#discussioncomment-2559547>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL7JQZ3FXMN3ZN4T2ULHJXTVE3B7DANCNFSM5TKSEMIQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
John A. Church
704.860.9189 Cell
